packages/eslint-plugin-tree-shaking/src/utils/helpers.ts

```
@@ -0,0 +1,134 @@
+
+  if (scope.upper && scope.upper.type !== 'global') return getLocalVariable(variableName, scope.upper);
+};
+
+const flattenMemberExpressionIfPossible = (node: Node): string | null => {
+  switch (node.type) {
+    case 'MemberExpression':
+      if (node.computed || node.property.type !== 'Identifier') {
+        return null;
+      }
+      // eslint-disable-next-line no-case-declarations
+      const flattenedParent = flattenMemberExpressionIfPossible(node.object);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)